images/db/migrate/20150430171341_translate_refinery_images.rb

```diff
@@ -1,22 +1,20 @@
 class TranslateRefineryImages < ActiveRecord::Migration[4.2]
-  def self.up
-    begin
-      ::Refinery::Image.create_translation_table!({
-        image_alt: :string,
-        image_title: :string
-      }, {
-        :migrate_data => true
-      })
-    rescue NameError
-      warn "Refinery::Image was not defined!"
-    end
-  end
+  def change
+    create_table :refinery_image_translations do |t|
+
+      # Translated attribute(s)
+      t.string :image_alt
+      t.string :image_title
 
-  def self.down
-    begin
-      Refinery::Image.drop_translation_table! migrate_data: true
-    rescue NameError
-      warn "Refinery::Image was not defined!"
+      t.string  :locale, null: false
+      t.integer :refinery_image_id, null: false
+
+      t.timestamps null: false
     end
+
+    add_index :refinery_image_translations, :refinery_image_id, name: :index_refinery_image_translations_on_refinery_image_id
```

Comment:
This index has been removed in the generators in the latest version of Mobility, see shioyama/mobility#198. I think you don't need this one (since it's covered by the last index below on the combo of `refinery_image_id` and `locale`).